source/includes/linux/common-installation.rst

```diff
@@ -4,8 +4,8 @@ The following tabs provide examples of installing MinIO onto 64-bit Linux
 operating systems using RPM, DEB, or binary. The RPM and DEB packages
 automatically install MinIO to the necessary system paths and create a
 ``systemd`` service file for running MinIO automatically. MinIO strongly
-recommends using RPM or DEB installation routes.
-
+recommends using RPM or DEB installation routes. To update deployments
```

Comment:
One of two proposed locations for the update link.

The title of the linked section references `systemctl` so I used that. Using both `systemd` and `systemctl` here seems a bit confusing, but so would calling it `systemd` and landing on a page about `systemctl`. Suggestions welcome.

Reply:
I think just calling it `systemctl` is fine since that is the proc that actually handles the automation.

Maybe for the graph on 4-6 we could state something like "create a ``minio`` service for ``systemctl``"

Reply:
Went with both your suggestions. And joined sentences since I was in there messing with existing text anyway.

What about the other instances of this same original paragraph? Edit or leave them alone?

Reply:
:thinking: if you want to expand scope and hit the other ones, go for it. Might as well while we're here, so to speak.